src/Mentat.UI/Controllers/CardController.cs

```diff
@@ -55,6 +55,29 @@ public ActionResult AddEditCard(string id = null)
         [ValidateAntiForgeryToken]
         public ActionResult SaveCard(string id, [FromForm] Card card, [FromForm] string tagList)
         {
+            // Checks if card provided is new or exists. If the card exists, checks owner changes.
+            Card oldCard;
+            if (id == null)
+            {
+                oldCard = null;
+            }
+            else
+            {
+                oldCard = _cardService.GetCard(id);
+                oldCard.Owner ??= card.Owner;
+            }
+            if ((oldCard != null) && (oldCard.Owner != card.Owner))
+            {
+                if (oldCard.IsPrivate)
```

Comment:
If this check is in place, you should add a true copy button somewhere that copies it over to the user. Why display a private card if I can't create a copy of it?